### ISSUE TITLE
Set up subscriptions 

### ISSUE THREAD

**Issue by lorensr** [apollographql/meteor-integration#36]
https://medium.com/apollo-stack/graphql-subscriptions-in-apollo-client-9a2457f015fb?source=linkShare-749064bd48ca-1473998135

**Comment by Siyfion** [comment]
This is really getting quite crucial to have this implemented now, any idea on when this might get looked at @lorensr ?

**Comment by lorensr** [comment]
Whenever someone is interested enough to do a pr! :)

**Comment by Siyfion** [comment]
Argh. I wasn't planning on spending a load of time on this.
But then I also have bugs that can only be fixed by getting subscriptions in and working... 😫

**Comment by Torsten85** [comment]
Hey there... I got subscriptions working by setting up a websocket server on top of meteor. But it didn't feel right because we already have a persistent ddp connection. So I wrote an alternative to subscription-transport-ws for meteor:

https://github.com/Torsten85/subscription-transport-meteor

Maybe this is of some use to you.

**Comment by jvallarvisp** [comment]
I was able to add susbcriptions. Before `WebApp.connectHandlers.use(Meteor.bindEnvironment(graphQLServer));`

Add this code, you also have to create 

`var subscriptionManager = require('./../imports/data/subscriptions').subscriptionManager;
    var server = http.createServer(graphQLServer);

    new SubscriptionServer({
        subscriptionManager
    }, server);

    server.listen(settings.apiPort, () => {
        console.log("Subscription manager running "  + settings.apiPort)
    })`

**Comment by Siyfion** [comment]
@jvallarvisp I've looked at your code, and you aren't using Meteor, or `meteor-integration`...?

**Comment by jvallarvisp** [comment]
@Siyfion your code for ` ./../imports/data/subscriptions` should be look like this
```
import { SubscriptionManager } from 'graphql-subscriptions'
import schema, { pubsub } from './schema'

const subscriptionManager = new SubscriptionManager({
  schema,
  pubsub,
  setupFunctions: {
    countUpdated: () => ({
      // Run the query each time count updated
      countUpdated: () => true
    })
  },
});

export { subscriptionManager, pubsub };
```

**Comment by xavcz** [comment]
Hey folks!

How are you doing subscriptions at the moment with `apollo` ?

I've opened a PR to make it easy, your feedbacks are more than welcomed: #85

**Comment by xavcz** [comment]
Nom nom nom, that's published! Be sure to be on `apollo@0.6.0` to give it a try. The docs will be updated soon (PR opened) 🙂

Side-note: I've had a pretty hard time with the test for subscriptions. If someone wants to give it a try, that would be great & I'd be happy to help!
